src/app/components/Curation/getComponentName/index.ts

```diff
@@ -21,12 +21,18 @@ const {
   HIERARCHICAL_CURATION_GRID,
   MOST_READ,
   NOT_SUPPORTED,
+  RADIO_SCHEDULE,
 } = COMPONENT_NAMES;
 
-export default (
-  visualStyle: VisualStyle,
-  visualProminence: VisualProminence,
-) => {
+export default ({
+  visualStyle,
+  visualProminence,
+  radioSchedule,
+}: CurationProps) => {
+  if (radioSchedule) {
+    return RADIO_SCHEDULE;
+  }
+
```

Comment:
Given that we don't technically need visualStyle & visualProminence to determine all curations, I am wondering if we might make all of the props to this function optional... 

To do so, we can use `Partial` type (search for other implementations in the repo and also the docs: https://www.typescriptlang.org/docs/handbook/utility-types.html#partialtype)

Doing this will also mean that we no longer need to pass props which were previously mandatory - e.g. in the `getComponentName` test, we can now only pass in a `radioSchedule` object, and omit `visualStyle` and `visualProminence`

 ```suggestion
export default ({
  visualStyle,
  visualProminence,
  radioSchedule,
}: Partial<CurationProps>) => {
  if (radioSchedule) {
    return RADIO_SCHEDULE;
  }

```

Reply:
A new thing to learn! Thanks. I have used Partial in the code now and only pass radioSchedule into getComponentName. Seems to work; all tests pass.